pytest.ini

```
@@ -1,11 +1,14 @@
 [pytest]
-addopts = --verbose -ra
-python_paths = src/ tests/
+addopts = --verbose -ra --runslow
+pythonpath = src/ tests/
 env =
     COGNITO_DEPLOYMENT_MODE=Development
-    SECRET_KEY=changme
     DEFAULT_X_GOPHISH_CONTACT=test@test.com
     LOCAL_API_KEY=somerandomkey
+    PYTESTING=True
     PYTEST_SETTINGS=1
```

This aligns with how things are sorted in other files.

```suggestion
    PYTEST_SETTINGS=1
    PYTESTING=True
```